playbooks/files/tigkstack/deploy_go.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+pushd /opt
+  wget -P /opt/ https://storage.googleapis.com/golang/go1.7.3.linux-amd64.tar.gz
```

Comment:
Updated to latest 1.8 however it doesn't look like go provides a "latest" link. If we can find one then I'd be happy to switch it up.